configure.sh

```diff
@@ -80,6 +80,9 @@ Optional Path to Optional Packages:
   --poly-dir=PATH          path to top level of LibPoly source tree
   --symfpu-dir=PATH        path to top level of SymFPU source tree
 
+Build limitations:
+  --lib-only               only build the library, but not the executable or the parser (default: off)
```

Comment:
Make sure this line is 80 chars.

Reply:
Thanks -- done in 085bf094d918953c803d8b76848996ae4178445e